src/Server/GRPCServer.h

```diff
@@ -35,6 +35,9 @@ class GRPCServer
     /// Returns the number of currently handled connections.
     size_t currentConnections() const;
 
+    /// TODO: Returns the number of current threads.
+    size_t currentThreads() const { return 1; }
```

Comment:
CC @vitlibar how to do it?

Reply:
I don't like to leave this kludge in code.

Reply:
There are a) 1 thread per connection + b) one thread which is always listening for new connections since start() is called. You don't want to take b) into account, do you?

If so the following very simple function will help

```
size_t GRPCServer::currentThreads() const
{
    return runner->getNumCurrentCalls();
}
```

Reply:
Then it's the same as `currentConnections()`

Reply:
> CC @vitlibar how to do it?

There are a) 1 thread per call + b) one utility thread which listens for new calls and run grpc queue to process calls. 

If a) is enough for the metrics you can add a simple function

```
size_t GRPCServer::currentThreads() const
{
    return runner->getNumCurrentCalls();
}
```